src/lib/stack.ts

```diff
@@ -15,6 +15,7 @@ export class Stack {
   constructor(client: AxiosInstance, config: StackConfig) {
     this._client = client;
     this.config = config;
+    this.config.live_preview = config.live_preview
```

Comment:
@harshithad0703 , can you remove the line as it seems redundant - the above live should take care of the live_preview property.

Reply:
Removed that line
Thank you